src/main/java/org/opentripplanner/middleware/utils/NotificationUtils.java

```diff
@@ -37,12 +42,35 @@
     public static final String FROM_PHONE = getConfigPropertyAsText("NOTIFICATION_FROM_PHONE");
     private static final String SPARKPOST_KEY = getConfigPropertyAsText("SPARKPOST_KEY");
     private static final String FROM_EMAIL = getConfigPropertyAsText("NOTIFICATION_FROM_EMAIL");
+    public static final String OTP_ADMIN_DASHBOARD_FROM_EMAIL = getConfigPropertyAsText("OTP_ADMIN_DASHBOARD_FROM_EMAIL");
 
     /**
-     * Send a SMS message to the provided phone number
-     * @param toPhone - e.g., +15551234
-     * @param body - SMS message body
-     * @return messageId if message was sucessful (null otherwise)
+     * Send templated SMS to {@link OtpUser}.
+     * @param otpUser       target user
+     * @param smsTemplate   template to use for SMS message
+     * @param data          template data
+     * @return              messageId if message was successful (null otherwise)
+     */
+    public static String sendSMS(OtpUser otpUser, String smsTemplate, Object data) {
+        if (!otpUser.isPhoneNumberVerified) {
+            LOG.error("Cannot send SMS to unverified user ({})!", otpUser.email);
+            return null;
+        }
+        try {
+            String body = TemplateUtils.renderTemplate(smsTemplate, data);
+            return sendSMS(otpUser.phoneNumber, body);
+        } catch (TemplateException | IOException e) {
+            // This catch indicates there was an error rendering the template. Note: TemplateUtils#renderTemplate
+            // handles Bugsnag reporting/error logging, so that is not needed here.
```

Comment:
The Bugsnag reference should be removed.

Reply:
Bugsnag is still used to report errors, so I think I'll keep this in place.